tools/expotools/src/commands/AndroidBuildPackages.ts

```diff
@@ -382,10 +390,19 @@ export default (program: any) => {
     .command('android-build-packages')
     .alias('abp')
     .description('Builds all Android AAR packages for Turtle')
-    .option('-s, --sdkVersion [string]', 'SDK version')
+    .option('-s, --sdkVersion [string]', '[optional] SDK version')
     .option(
       '-p, --packages [string]',
       '[optional] packages to build. May be `all`, `suggested`, or a comma-separate list of package names.'
     )
-    .asyncAction(action);
+    .asyncAction(async (options: Partial<ActionOptions>) => {
+      const sdkVersion =
+        options.sdkVersion ?? (await ProjectVersions.getNewestSDKVersionAsync('android'));
+
+      if (!sdkVersion) {
+        throw new Error('Could not infer SDK version, please run with `--sdkVersion SDK_VERSION`');
+      }
+
+      await action({ sdkVersion, ...options });
```

Comment:
You probably wanted to do
```suggestion
      await action({ ...options, sdkVersion });
```
otherwise `options.sdkVersion` would still be `undefined` if not passed 😉

Reply:
Fixed in https://github.com/expo/expo/pull/9983/commits/e97bcfee23efac69bc729c4631c84cf9ee5b6cbf! Good catch!